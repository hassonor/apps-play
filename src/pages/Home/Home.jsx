import GoogleAppsResults from "../../components/googleApps/GoogleAppsResults";
import GoogleAppsSearch from "../../components/googleApps/GoogleAppsSearch";

function Home() {
    return (
        <>
            <GoogleAppsSearch/>
            <GoogleAppsResults/>

        </>
    )
}

export default Home