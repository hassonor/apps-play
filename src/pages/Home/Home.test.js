import React from "react";
import {shallow} from "enzyme";
import Home from "./Home";

it("expect to render Home Page", () => {
    expect(shallow(<Home/>)).toMatchSnapshot();
});