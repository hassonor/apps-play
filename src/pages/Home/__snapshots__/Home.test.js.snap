// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Home Page 1`] = `
<Fragment>
  <GoogleAppsSearch />
  <GoogleAppsResults />
</Fragment>
`;
